gym_homer/envs/homer_env.py

```diff
@@ -229,39 +248,42 @@ def _update_battery(self, solar, loads, max_d, max_c, action
         # Calculate Grid State, Update Battery state
```

Comment:
I think we should remove the "discrete" code path.  We should model the "discrete max" case as the fractional with an interval of 1.  
ex: 
```interval = 1
action space = 2*1 +1 = 3   -> [0,1,2]

// in step() the action is then modeled as 
self.updated_action = (action - self.action_intervals) / self.action_intervals
updated_action = [(0-1) /1 ,(1-1) /1, (2-1) /1] = [-1,0,1]

// functionally [max_discharge, no action, max_charge]
```
Maintaining two code paths is asking for bugs (we've already seen a couple)

Reply:
agreed.